tools/dev/import-dev-keys.sh

```diff
@@ -13,47 +11,3 @@ for key in $KEYBASE_KEY_URLS; do
   echo Importing $key...
   \curl -sSL $key | gpg --quiet --no-auto-check-trustdb --import -
 done
-
```

Comment:
Wait are we even still using KeyBase URLs? I deleted my account since I didn't think anything was using it of importance and last I heard KeyBase hasn't been doing much updates if any.

Reply:
¯\\\_(ツ)_/¯ I'm just fixing the script.

> Should probably be updated to include GitHub keys (e.g., https://github.com/smcintyre-r7.gpg).

Reply:
cc @smcintyre-r7